packages/core/src/extensions/SideMenu/SideMenuPlugin.ts

```diff
@@ -192,6 +192,12 @@ export class SideMenuView<
       true,
     );
 
+    // Update state when the editor scrolls
+    this.pmView.root.addEventListener(
```

Comment:
Makes sense, though we already attach an existing `onScroll` handler a few lines below - wouldn't it make more sense to call `this.updateStateFromMousePos` there instead of creating an additional listener?

Reply:
I didn't see that one! Very true